proto/assessment.proto

```diff
@@ -39,6 +39,13 @@ option go_package = "api/assessment";
 // Representing the link between orchestrator and discovery: Assessing evidences
 // from discovery and sending results to orchestrator
 service Assessment {
+  // Configure assessment, e.g. setting up connections to other services
+  rpc Configure(ConfigureAssessmentRequest) returns (ConfigureAssessmentResponse) {
+    option (google.api.http) = {
+      post : "/v1/assessment/start"
```

Comment:
URL configure instead of start? What is the idea of this RPC anyway?

Reply:
I would rather tread the "Configuration" as a resource, meaning that we will have at least a `GetConfiguration" and `UpdateConfiguration` call that maps to /v1/assessment/configuration GET/PUT

Reply:
First I called it start because I wanted to init the stream to evidence store separately from `AssessEvidence`. Then we decided to do it in `AssessEvidence` which is no problem. But we now have the problem that all Services are connecting to localhost:9000 but in MEDINA we have different services (instead of one engine.go) and, therefore, different ports. The idea of Configure is to decide that in real time. 

As of now, in the medina project, @garuppel changes the external clouditor code (library) to tackle this problem (set the ports right). We wanted to change in that in the next PR.

The URL I have to change, ofc.